web/src/lib/utils/helpers.ts

```diff
@@ -144,6 +144,18 @@ export const getUserNameOrEmail = (userData: UserData | undefined): string => {
   }
 };
 
+export const getNavBarBusinessTitle = (userData: UserData | undefined): string => {
```

Comment:
This function should be tested. Also, I think it should not live here in "helpers" but rather in its own thing. It is not a generic-use helper function, it has a very specific purpose for the NavBar only (as the title suggests)

Reply:
This would proabbly live in the lib/utils not components/navbar right?

Reply:
Also when you say this function should be tested. Initially I thought you meant in its own test file in utils but on second thought I think you probably mean in the NavMenuPopup test file implicitly. Is that correct?

Reply:
When you say this should be tested. Do you mean impliclty in the tests for the NavBarPopupMenu? or do you mean explictly in its own test file? (Wouldn't testing in its own file be a bit of a weird practice since its just an individual function and we talked about how you don't rip out functions to test them alone?)